lang/cs/Org.Apache.REEF.Network/Group/Operators/IReduceSender.cs

```diff
@@ -26,7 +26,7 @@ namespace Org.Apache.REEF.Network.Group.Operators
     public interface IReduceSender<T> : IMpiOperator<T>
     {
         /// <summary>
-        /// Sends data to the operator's ReduceReceiver to be aggregated.
+        /// Get reduced data from children, reduce with the data given, then sends reduced data to parent
```

Comment:
This new comment only makes sense from perspective of TreeTopology.  I wouldn't change the public API comments based on the implementation of ReduceSender.  I imagine most people still view things logically in terms of Flat Topologies